app/uk/gov/hmrc/partnershipidentification/connectors/RegisterWithMultipleIdentifiersConnector.scala

```diff
@@ -29,7 +29,7 @@ class RegisterWithMultipleIdentifiersConnector @Inject()(http: HttpClient,
                                                          appConfig: AppConfig
                                                         )(implicit ec: ExecutionContext) {
 
-  def register(jsonBody: JsObject
+  def register(jsonBody: JsObject,regime: String
```

Comment:
Tiny comment,  but could you format this please? Looks like there's a space missing.